std/node/fs_test.ts

```diff
@@ -45,3 +48,52 @@ test(function readFileEncodeUtf8Success() {
   assertEquals(typeof data, "string");
   assertEquals(data as string, "hello world");
 });
+
+if (Deno.build.os !== "win") {
+  run({
```

Comment:
You don't really need `run` for system dependent tests here. You can just directly use APIs under `Deno` namespace for creating the links and removing them

Reply:
Make sence, I use Deno.symlinkSync() for creating the links.

Reply:
Just for now, until we implement symlink for Windows.